int16 my_pos_x
int16 my_pos_y
int16 enemy1_x
int16 enemy1_y 
int16 enemy2_x
int16 enemy2_y
int16 ally_x
int16 ally_y
int16 goal_pos_x
int16 goal_pos_y
---
int16 next_pos_x
int16 next_pos_y
